clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ServerGroupParameters.groovy

```diff
@@ -105,15 +115,24 @@ class ServerGroupParameters {
    */
```

Comment:
We should add a TODO and reference the openstack ticket that will address this issue in both circumstances.